CalendarKit-Swift/SwiftCal/ICSEventParser.swift

```diff
@@ -21,6 +21,7 @@ class ICSEventParser: NSObject {
         static let organizer2 = "ORGANIZER:"
         static let sequence = "SEQUENCE:"
         static let location = "LOCATION:"
+        static let location2 = "LOCATION;"
```

Comment:
Can we name this something descriptive like `locationPrefix` or something? Maybe look at the spec and see what they call it?

Reply:
I just followed the pattern from a couple other fields that handle this situation to keep the codebase consistent. The parser could use some refactoring to DRY out the parsing, and significant work to handle edge cases correctly and make parsing more efficient. I'm filing a ticket to track that.

[The standard](https://tools.ietf.org/html/rfc5545) calls the pieces `name`, `param`s, and `value` btw.